README.md

```diff
@@ -57,5 +119,43 @@ for dom,prob in DOMAINS['acrobatics']:
 # Access all the benchmarks for a specific collection
 for domain_name in COLLECTIONS['all-fond-papers']:
     print(domain_name)
+```
+
+Script can also be run as follows:
 
+```python
+$ python catalogue.py
+(  8) acrobatics
+( 11) beam-walk
+( 30) blocksworld
+( 15) blocksworld-2
+( 50) blocksworld-new
+(  1) bus-fare
+...
+...
+...
+All domains in collection 'interesting':
+         climber
+         bus-fare
+         river
 ```
+
+## FOND Planners
+
+Using PDDL with `oneof` construct:
+
+- [PR2](https://github.com/QuMuLab/pr2) ([2024](https://arxiv.org/pdf/2312.11675)): Next iteration of PRP. 😉
+- [CFOND-ASP](https://github.com/ssardina-research/cfond-asp) ([2023](https://ojs.aaai.org/index.php/ICAPS/article/view/19789/19548))
+- [Paladinus](https://github.com/ramonpereira/paladinus) ([2022](https://ojs.aaai.org/index.php/ICAPS/article/view/19789/19548)).
+- [FOND-SAT](https://github.com/tomsons22/FOND-SAT) ([2018](https://arxiv.org/pdf/1806.09455))
+- [MyND](https://github.com/JackyCSer/MyNDPlanner) ([2010](https://ojs.aaai.org/index.php/ICAPS/article/view/13408/))
+- [PRP](https://github.com/QuMuLab/planner-for-relevant-policies) ([2012](https://cdn.aaai.org/ojs/13520/13520-40-17038-1-2-20201228.pdf))
+- FIP ([2011](https://www.ijcai.org/Proceedings/11/Papers/326.pdf))
```

Comment:
FIP didn't accept `oneof`. Needs the domains to be manually determinized first (with specific naming on outcomes (which must number fewer than 10 :P )). Move this to the "Not using oneof" section?

Reply:
ohhh didn't know this. I don't think I've ever had the FIP system/code; is it anywhere? But for what you say makes sense given some domains I have seen that had determinized actions with special syntax. 

Definitively, we should move it to "didn't accept oneof" section. Given that you know exactly what it needed, it would be helpful to state it so as well (though again not sure if one can grab the code). Will do the change now.

Reply:
I think they sent the source to me at one point -- I remember recompiling to handle >9 outcomes, but can't remember where the source was specifically.

All looking good now!